Cargo.toml

```diff
@@ -68,19 +68,19 @@ alloy-transport-http = { version = "0.2", path = "crates/transport-http", defaul
 alloy-transport-ipc = { version = "0.2", path = "crates/transport-ipc", default-features = false }
 alloy-transport-ws = { version = "0.2", path = "crates/transport-ws", default-features = false }
 
-alloy-core = { version = "0.7.7", default-features = false }
-alloy-dyn-abi = { version = "0.7.7", default-features = false }
-alloy-json-abi = { version = "0.7.7", default-features = false }
-alloy-primitives = { version = "0.7.7", default-features = false }
-alloy-sol-types = { version = "0.7.7", default-features = false }
+alloy-core = { version = "0.8.0", default-features = false }
+alloy-dyn-abi = { version = "0.8.0", default-features = false }
+alloy-json-abi = { version = "0.8.0", default-features = false }
+alloy-primitives = { version = "0.8.0", default-features = false }
+alloy-sol-types = { version = "0.8.0", default-features = false }
 
 alloy-rlp = { version = "0.3", default-features = false }
 
 alloy-chains = { version = "0.1.18", default-features = false }
 
 # ethereum
-ethereum_ssz_derive = "0.5"
-ethereum_ssz = "0.5"
+ethereum_ssz_derive = "0.7"
+ethereum_ssz = "0.7"
```

Comment:
blocked on ssz 0.8

Reply:
why?

Reply:
nvm, missing impls tho

Reply:
sigh

Reply:
can workaround by just using Bloom::new() since it's implemented for FixedBytes<N> but not Bloom(FixedBytes)

Reply:
> sigh
Need `Bloom` impl

Reply:
@yash-atreya mind porting the previous alloy-core impl to ethereum-ssz ?

Reply:
not a blocker tho